database/mysql_db.py

```diff
@@ -1,50 +1,51 @@
-import time
-from typing import Optional
+from __future__ import annotations
 
-from global_logger import logged_method
+from typing import List
 
+import mysql.connector
+from mysql.connector import MySQLConnection
+from mysql.connector.cursor import MySQLCursor, \
+    CursorBase, MySQLCursorBuffered, MySQLCursorRaw, MySQLCursorBufferedRaw, \
+    MySQLCursorDict, MySQLCursorBufferedDict, MySQLCursorNamedTuple, MySQLCursorBufferedNamedTuple, MySQLCursorPrepared
 
-# TODO: change MysqlDB to db and change query builder to mysql_query
-class MysqlDB:
+from database.db import QueryBuilder, DBContext, DBContextManager, DBTransaction, DBRollbackError, DBCommitError, \
+    DBQueryError, DB
+from global_logger import *
+
+
+class MySQLQueryBuilder(QueryBuilder):
```

Comment:
Because all this class does is build queries for use with MySQL, I renamed it `MySQLQueryBuilder`, and so we can create different implementations I made an abstract base class `QueryBuilder` based off the methods defined in this class.